server/entitymanager/entitymanager.go

```diff
@@ -354,3 +354,60 @@ func New(chassisConfigFile string) (*InMemoryEntityManager, error) {
 
 	return newManager, nil
 }
+
```

Comment:
Hi Liam, just letting you know we are working on importing this code into Google internally, which has strict code quality standards. I'll be leaving comments based on what needs to be updated such that the code can pass our internal linters.

You may notice some of the already submitted code already violates the comments I leave, I'll be going through and gradually fixing everything up.

Reply:
Thank you, please let me know if there is anything else I need to change before this can go through!